apps/alchemist/test/integration/performance/cve_test.exs

```diff
@@ -0,0 +1,79 @@
+defmodule Alchemist.Performance.CveTest do
```

Comment:
Forgot Valkyrie had this... Does it work in alchemist?

Reply:
It was tagged as :skip so it never ran. Forgot to manually attempt it, I'll try that now.

Reply:
Discussed in a DM: Removed